docs/deployment/remote-docker-gateway.md

```diff
@@ -44,7 +44,7 @@ infrastructure_targets:
     remote_gateway_url: 'http://1.2.3.4:7105/pub'
     remote_gateway_token: '5tr0nG_PA55VoRD'
 
-docker: 
+jobtype_extra:
```

Comment:
This YAML is not the job's manifest, but the configuration of the plugin, so in this case this field is valid:
```suggestion
docker:
```